.github/workflows/ci.yaml

```diff
@@ -349,6 +349,5 @@ jobs:
         with:
           context: .
           push: true
-          platforms: linux/amd64,linux/arm64,linux/arm64/v8
```

Comment:
I added this last night, and the GitHub Actions builders aren't able to cross-compile the images.  This *should* work, but it's not, and because of emulation, it's really slow.  So I'm going to give up on it, since staging/prod doesn't need it.